pkg/minikube/image/cache.go

```diff
@@ -60,7 +60,7 @@ func SaveToDir(images []string, cacheDir string) error {
 			dst := filepath.Join(cacheDir, image)
 			dst = localpath.SanitizeCacheDir(dst)
 			if err := saveToTarFile(image, dst); err != nil {
-				glog.Errorf("save image to file %q -> %q failed: %v", image, dst, err)
+				glog.Errorf("save image to file %q -> %q failed: the image doesn't exist", image, dst)
```

Comment:
thank you for taking this task,  
this will changes the error log, this doesn't seem to solve the example in the issue.

I would rather this instead of erroring logs give a solution message to the user.

please make sure to run the example in the issue and paste the Before/After output in the PR description.